Mathlib/Analysis/InnerProductSpace/LaxMilgram.lean

```diff
@@ -78,13 +78,13 @@ theorem ker_eq_bot (coercive : IsCoercive B) : ker B♯ = ⊥ := by
   exact antilipschitz.injective
 #align is_coercive.ker_eq_bot IsCoercive.ker_eq_bot
 
-theorem closed_range (coercive : IsCoercive B) : IsClosed (range B♯ : Set V) := by
+theorem isClosed_range (coercive : IsCoercive B) : IsClosed (range B♯ : Set V) := by
```

Comment:
did you deprecate this one?

Reply:
Ah no, I overlooked it. Thanks for spotting, fixed now.